lib/components/util/dropdown.tsx

```diff
@@ -180,10 +172,9 @@ const Dropdown = ({
         aria-expanded={open}
         aria-haspopup="listbox"
         aria-label={label}
-        as={!locale ? 'button' : ''}
+        as={nav ? '' : 'button'}
```

Comment:
These should always be buttons. Indeed, right now the navbar dropdown (language and account) buttons render into `<a ... role="button">`. See if you can get rid of the `nav` prop by writing instead `<Dropdown as="li"` (or `"span"`)?

Reply:
Good point about the button! I have removed the above line in [8f77197](https://github.com/opentripplanner/otp-react-redux/pull/897/commits/8f77197c5c9c1808fa359627deb383facf91a36c) and refactored the CSS to match. 

I changed the outer DropdownContainer to a traditional html element because the `SortResultsDropdown` [needs to pass a `className` to a DOM element](https://styled-components.com/docs/basics#styling-any-component) in order to work. Even if it was changed back to a styled component, passing `as` to `SortResultsDropdown` unfortunately doesn't work. 

I can't think of a way to conditionally render that container as either a `span` or `li` other than using props, but please let me know if I'm missing something!

Reply:
That's it!!! This works great, thank you so much!